### ISSUE TITLE
Error when running v1.1.4 locally

### ISSUE THREAD

**Issue by ff6347** [technologiestiftung/giessdenkiez-de-dwd-harvester#42]
@sebastian-meier I'm trying to run the harvester locally, since we are planning to have a workshop and I need to move this to something easier then AWS Fargate. When running the script I get the following error.
 
```plain
harvester_1  | FileNotFoundError: [Errno 2] No such file or directory: '/temp/trees-p2.csv'
harvester_1  | DEBUG:s3transfer.utils:Releasing acquire 0/None
harvester_1  | Traceback (most recent call last):
harvester_1  |   File "harvester.py", line 360, in <module>
harvester_1  |     s3.upload_file(path + "trees-p{}.csv".format(i + 1), os.getenv("S3_BUCKET"), "trees-p{}.csv".format(i + 1))
```

Any idea where this might come from? In production these files exists and the harvester works.

**Comment by sebastian-meier** [comment]
Do you have actual trees in your local database (see lines 313 and following)?

**Comment by ff6347** [comment]
Yes I have. What I didn't have was the radolan_geometry. I copied everything from the production DB and now it works.

**Comment by ff6347** [comment]
How did you create the content from that table?

**Comment by ff6347** [comment]
And there was no radolan_data created nor does the weather.geojson contain features

**Comment by sjockers** [comment]
I ran into the same problem when setting this up for Münster. I'm wondering how the radolan_geometry column is supposed to be populated? If this is automated, when/where in the process is this happening?

**Comment by JensWinter** [comment]
You mean the radolan_geometry table?
I think this is a manual one-time initialization step. There is a sql script in the [postgres-api repo](https://github.com/technologiestiftung/giessdenkiez-de-postgres-api/blob/master/sql/radolan_geometry.sql). It contains the geometry for Berlin, though.

If I remember correctly I used the [prepare.py](https://github.com/technologiestiftung/giessdenkiez-de-dwd-harvester/blob/master/harvester/prepare.py) script to create the buffer shape for Magdeburg and translated that into sql insert statement with the help of QGis.

**Comment by JensWinter** [comment]
Sure! I'm currently on vacation. So I'm gonna do that when I'm back home.

**Comment by ff6347** [comment]
Awesome. Yes please. Any documentation about that topic is very welcome. I guess it also would be better to have it within this repo here and not in the API.
